target/linux/pistachio/config-4.4

```diff
@@ -32,6 +32,7 @@ CONFIG_BLK_DEV_BSG=y
 CONFIG_BLK_DEV_DM=y
 CONFIG_BLK_DEV_DM_BUILTIN=y
 CONFIG_BLK_DEV_IO_TRACE=y
+CONFIG_BLK_DEV_LOOP=y
```

Comment:
This is getting removed if you do make kernel_menuconfig and save & exit. 
I think this is dependent on CONFIG_BLK_DEV that's why its getting removed.